worker/tern_worker.js

```diff
@@ -177,7 +246,7 @@ handler.analyze = function(value, ast, callback, minimalAnalysis) {
     };
     addTernFile(this.path, value);
 
-    architectResolver.onReady(function() {
+    architectResolver && architectResolver.onReady(function() {
```

Comment:
It is possible to not use architectResolver tern-plugin. The code there within the ternWorker event has reference to the object.
It is more safe to guard it